### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ovari**
Hopefully all the resources to translate now have direct links.

Removed the table to have a consistent style.

Few other improvements.

Thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,40 +3,26 @@
...
+Would you like to see Xournal++ in your own language? Translators are welcome to contribute to Xournal++.
 
-- Czech
-- Polish
-- Chinese
+Translation resources are in the following locations:
+* [Crowdin](https://crowdin.com/project/xournalpp/)
+* [desktop/com.github.xournalpp.xournalpp.appdata.xml](desktop/com.github.xournalpp.xournalpp.appdata.xml)
+* [desktop/com.github.xournalpp.xournalpp.desktop](desktop/com.github.xournalpp.xournalpp.desktop)
+* [desktop/com.github.xournalpp.xournalpp.xml](desktop/com.github.xournalpp.xournalpp.xml)
```
Comment (Technius) [inline README.md:16]: I think a slightly more explicit version reads better:
```suggestion
You can contribute translations to the following locations:
* [Crowdin](https://crowdin.com/project/xournalpp/)
* Submit a pull request on GitHub to change:
  * [desktop/com.github.xournalpp.xournalpp.appdata.xml](desktop/com.github.xournalpp.xournalpp.appdata.xml) (
  * [desktop/com.github.xournalpp.xournalpp.desktop](desktop/com.github.xournalpp.xournalpp.desktop)
  * [desktop/com.github.xournalpp.xournalpp.xml](desktop/com.github.xournalpp.xournalpp.xml)
```

File: README.md [hunk 2]
```diff
@@ -79,46 +65,46 @@ According to the Apple App Store guidelines, it is prohibited to publish unstabl
...
-<img src="https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile/-/raw/master/fastlane/metadata/android/en_US/images/tenInchScreenshots/03.png" width=100% title="Xournal++ Mobile Screenshot on Chromium OS"/>
+<img src="https://gitlab.com/TheOneWithTheBraid/xournalpp_mobile/-/raw/master/fastlane/metadata/android/en_US/images/tenInchScreenshots/03.png" width=100% title="Xournal++ Mobile on Chromium OS"/>
 
 </td></tr><tr><td>
 
 ## Toolbar / Page Background / Layer
 
 Multiple page background, easy selectable on the toolbar
-<img src="readme/background.png" width=100% title="Xournal++ Screenshot"/>
+<img src="readme/background.png" width=100% title="Xournal++"/>
```
Comment (Technius) [inline README.md:95]:
```suggestion
<img src="readme/background.png" width=100% title="Changing the page background in Xournal++"/>
```

File: README.md [hunk 3]
```diff
@@ -79,46 +65,46 @@ According to the Apple App Store guidelines, it is prohibited to publish unstabl
...
 
 </td><td>
 
 ## Layer sidebar and advanced layer selection
 
-<img src="readme/layer.png" width=100% title="Xournal++ Screenshot"/>
+<img src="readme/layer.png" width=100% title="Xournal++"/>
```
Comment (Technius) [inline README.md:101]:
```suggestion
<img src="readme/layer.png" width=100% title="Layers in Xournal++"/>
```

File: README.md [hunk 4]
```diff
@@ -79,46 +65,46 @@ According to the Apple App Store guidelines, it is prohibited to publish unstabl
...
 
 </td></tr><tr><td>
 
 ## Multiple predefined and fully customizable toolbars
 
-<img src="readme/toolbar.png" width=100% title="Xournal++ Screenshot"/>
+<img src="readme/toolbar.png" width=100% title="Xournal++"/>
```
Comment (Technius) [inline README.md:107]:
```suggestion
<img src="readme/toolbar.png" width=100% title="Custom toolbars in Xournal++"/>
```

**Review by Technius — COMMENTED** [review]
Thanks, the changes look reasonable to me. I have a few more suggestions (see above comments).

**Comment by ovari** [comment]
Thank you @Technius for your suggestions.

Hopefully all your suggestions have all been accepted. If not, please advise; otherwise, can you please merge?

Thank you

**Pull request merged**